### PR TITLE
Updated the definition of Azure Subscription

### PR DESCRIPTION
**Pull request by Ananyojha**
Added the line *Billing and Access Control Boundary*. This helps us better understand the use of Azure Subscription.  Billing - hints that Subscription are used for Billing purpose.  Access Control -  Indicates that unless you are authenticated at subscription, you aren't permitted to create a resource.

### CODE REVIEW

**Comment by cmcclister** [comment]
Learn Build status updates of commit _[d13cd05](https://github.com/Ananyojha/cloud-adoption-framework/commits/d13cd05332b0ba4b713314c77492dd18546928b6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/ready/considerations/fundamental-concepts.md](https://github.com/Ananyojha/cloud-adoption-framework/blob/patch-1/docs/ready/considerations/fundamental-concepts.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/0286ad65-6ec8-0c70-0cae-2edc1504c701/202211131351009317-1001/BuildReport?accessString=45e95fa231c263eb84cc9a60e36911cd91c8e5f93cda0e4ee15857b60ac4b98a).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/0286ad65-6ec8-0c70-0cae-2edc1504c701?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by Court72** [comment]
@alexbuckgit

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Ananyojha** [comment]
#sign-off

The proposed changes looks fine to me.  Thanks for reviewing my Pull Request

File: docs/ready/considerations/fundamental-concepts.md [hunk 1]
```diff
@@ -19,7 +20,7 @@ In this article, learn about fundamental concepts, terms used in Microsoft Azure
 It's helpful to know the following definitions as you begin your Azure cloud adoption journey:
 
 - **Resource:** An entity that's managed by Azure. Examples include Azure Virtual Machines, virtual networks, and storage accounts.
-- **Subscription:** A logical container for your resources. Each Azure resource is associated with only one subscription. Creating a subscription is the first step in adopting Azure.
+- **Subscription:** A Billing and Access control boundary. Azure Subscription works as logical container for your resources. Each Azure resource is associated with only one subscription. Creating a subscription is the first step in adopting Azure.
```
Comment (ckittel) [inline docs/ready/considerations/fundamental-concepts.md:23]: Keeps consistency with the rest, and fixes the typos (excessive capitalization)

```suggestion
- **Subscription:** A logical container for your resources. Each Azure resource is associated with only one subscription. Creating a subscription is the first step in adopting Azure. A subscription serves as a billing and access control boundary.
```

**Review by ckittel — CHANGES_REQUESTED** [review]

**Comment by ckittel** [comment]
#please-close

**Pull request closed without merging**